.travis/debian10/run.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+docker exec cpp_tango mkdir -p /home/tango/src/build
+
+echo "Build cppTango:$CMAKE_BUILD_TYPE"
+
+docker exec cpp_tango cmake            \
+  -H/home/tango/src                    \
+  -B/home/tango/src/build              \
+  -DCMAKE_VERBOSE_MAKEFILE=true        \
+  -DCPPZMQ_BASE=/home/tango            \
+  -DCMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE
+
+if [ $? -ne "0" ]
+then
+    exit -1
```

Comment:
Consider just adding `set -e` to the top of the file.

Reply:
Done.